tests/helpers/utils.ts

```diff
@@ -136,7 +141,7 @@ export async function createAddon({
     console.debug(`\tember ${emberCliArgs.join(' ')}`);
   }
 
-  let result = await execa('ember', emberCliArgs, {
+  let result = await execa('npx', ['ember-cli@beta', ...emberCliArgs], {
```

Comment:
without this change, we use ember-cli-typescript with `--typescript`, which we don't want, as that process is broken (to over abbreviate)

here is the error: https://github.com/embroider-build/addon-blueprint/actions/runs/5801417360/job/15725641685#step:5:348